packages/webamp/.babelrc

```diff
@@ -18,7 +18,13 @@
         }
       }
     ],
-    "@babel/preset-react",
+    // TODO: Can TypeScript just do this for us?
```

Comment:
Hi @captbaritone, yes! This is coming in TS 4.1 https://devblogs.microsoft.com/typescript/announcing-typescript-4-1-beta/#jsx-factories :)

Reply:
I saw that! I'm starting to play with it here: https://github.com/captbaritone/webamp/pull/1031